### PR TITLE
remove handle references from api's

### PR DESCRIPTION
**Pull request by danewalton**

### CODE REVIEW

File: source/include/azure_iot_hub_client.h [hunk 1]
```diff
@@ -189,7 +190,7 @@ typedef struct AzureIoTHubClient
         const uint8_t * pucAzureIoTHubClientSymmetricKey;
         uint32_t ulAzureIoTHubClientSymmetricKeyLength;
 
-        uint32_t ( * pxAzureIoTHubClientTokenRefresh )( struct AzureIoTHubClient * pxAzureIoTHubClientHandle,
+        uint32_t ( * pxAzureIoTHubClientTokenRefresh )( struct AzureIoTHubClient * pxAzureIoTHubClient,
```
Comment (hihigupt) [inline source/include/azure_iot_hub_client.h:193]: AzureIoTHubClient_t pxAzureIoTHubClient

File: source/include/azure_iot_provisioning_client.h [hunk 2]
```diff
@@ -24,7 +24,8 @@
 
 #define azureiotPROVISIONING_RESPONSE_MAX   ( azureiotTOPIC_MAX + azureiotPROVISIONING_REQUEST_PAYLOAD_MAX )
 
-typedef struct AzureIoTProvisioningClient * AzureIoTProvisioningClientHandle_t;
+/* Forward declaration of Azure IoT Provisioning Client */
+typedef struct AzureIoTProvisioningClient AzureIoTProvisioningClient_t;
```
Comment (hihigupt) [inline source/include/azure_iot_provisioning_client.h:29]: Why do you need frwd declaration?

**Comment by hihigupt** [comment]
>         uint32_t ( * pxTokenRefresh )( struct AzureIoTProvisioningClient * pxAzureIoTProvisioningClientHandle,

should we remove handle from this too?

---
Refers to: source/include/azure_iot_provisioning_client.h:67 in a78b6da. [](commit_id = a78b6da072f65b28949b8f873e22dd3feed1e00b, deletion_comment = False)

**Review by hihigupt — APPROVED** [review]

File: source/include/azure_iot_provisioning_client.h (hunk 2 above)
Comment (danewalton) [reply source/include/azure_iot_provisioning_client.h:29]: nope not needed. my error

**Pull request merged**